lego/utils/youtube_validator.py

```diff
@@ -0,0 +1,10 @@
+from django.core.validators import RegexValidator
+
+youtube_validator = RegexValidator(
```

Comment:
Is this maybe a bit too strict? What if youtube changes their URLs a bit in the future?  
Is it not enough to just check that the domain is `youtube.com` or `youtu.be`?
Why are you using non-capturing groups? `(?:`

Reply:
I want the regex to match against the id of the youtube video as well, since we don't want video urls without ids 🤔

I use non-capturing groups, as I only want it to match the regex and not extract the result from the groups later. Not really sure if this is a good way to implement it, however. 😅